.travis.yml

```diff
@@ -20,7 +20,7 @@ before_script:
   - export COVERALLS_SERVICE_NAME=travis-ci
 
 env:
-  - DO_COVERAGE="coverage" BUILD_OPT="-DNATS_COVERAGE=ON -DNATS_BUILD_ARCH=64 -DNATS_BUILD_TYPE=Debug -DNATS_COVERAGE_UPLOAD=ON -DNATS_BUILD_TLS_USE_OPENSSL_1_1_API=ON -DCMAKE_BUILD_TYPE=Debug"
+  - DO_COVERAGE="coverage" BUILD_OPT="-DNATS_COVERAGE=ON -DNATS_BUILD_ARCH=64 -DCMAKE_BUILD_TYPE=Debug -DNATS_COVERAGE_UPLOAD=ON -DNATS_BUILD_TLS_USE_OPENSSL_1_1_API=ON -DCMAKE_BUILD_TYPE=Debug"
```

Comment:
I had both `-DNATS_BUILD_TYPE` and `-DCMAKE_BUILD_TYPE` on that line (proof that was clearly not the right approach, but I did not know how to do..). Could you remove say the instance at the end of the line? Thanks

Reply:
Whoops, missed that. Sorry. Fixed.